libexec/helmenv

```diff
@@ -41,7 +41,7 @@ abs_dirname() {
 }
 
 if [ -z "${HELMENV_ROOT}" ]; then
-  HELMENV_ROOT="${HOME}/.helmenv"
+  HELMENV_ROOT="$(cd "$(dirname "$(readlink_f "${0}")")/.." && pwd)"
```

Comment:
> readlink_f

What's this?
`readlink -f`?

Reply:
To be honest, I copy pasted it from tfenv 😄

Reply:
Fixed!